src/components/ContentNode/Table.vue

```diff
@@ -44,7 +64,7 @@ export default { name: 'Table' };
   th {
     border-color: var(--color-fill-gray-tertiary);
     border-style: solid;
-    border-width: 1px 0;
+    border-width: var(--table-border-width, 1px 1px);
```

Comment:
Sam wanted to add vertical borders by default to tables now, so I did that but thought it might make sense to expose this as a CSS var, so third party clients can modify it.